tests/test_cmab_meta_model.py

```
@@ -228,6 +241,59 @@ def test_backbone_update_changes_backbone_and_heads(
         assert any(not np.allclose(b, a) for b, a in zip(w_before, meta.backbone.weight_arrays))
         assert not np.allclose(mu_before, self._head_mu(meta, "a"))
 
+    def test_backbone_anchor_zero_matches_unanchored(self, rng: np.random.Generator) -> None:
+        """``backbone_anchor_strength=0.0`` (explicit) reproduces the default unanchored behavior exactly."""
+        actions, rewards, context = self._balanced_batch(rng)
+        default_meta = self._build_meta(with_backbone=True)
+        zero_meta = self._build_meta(with_backbone=True, backbone_anchor_strength=self.anchor_strength_disabled)
+        default_meta.update(actions=actions, rewards=rewards, context=context)
+        zero_meta.update(actions=actions, rewards=rewards, context=context)
+        for w_default, w_zero in zip(default_meta.backbone.weight_arrays, zero_meta.backbone.weight_arrays):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)